csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -842,13 +911,17 @@ public Settings(bool formatDefaultValues, TypeRegistry typeRegistry) : this(form
             /// <param name="formatDefaultValues"><c>true</c> if default values (0, empty strings etc) should be formatted; <c>false</c> otherwise.</param>
             /// <param name="typeRegistry">The <see cref="TypeRegistry"/> to use when formatting <see cref="Any"/> messages. TypeRegistry.Empty will be used if it is null.</param>
             /// <param name="formatEnumsAsIntegers"><c>true</c> to format the enums as integers; <c>false</c> to format enums as enum names.</param>
+            /// <param name="indent">The indentation string to use for multi-line formatting. <c>""</c> to disable multiline format.</param>
             private Settings(bool formatDefaultValues,
                             TypeRegistry typeRegistry,
-                            bool formatEnumsAsIntegers)
+                            bool formatEnumsAsIntegers,
+                            string indent = "")
```

Comment:
Presumably this is a string rather than an integer number of spaces to allow for tab indentation? (I think it's probably the right call - I'm just interested.)

Reply:
Yes exactly. This was to cover for `\t`, but I also used it during development/hacking. Printed `>` instead of spaces for easier visual validation :-)